### PR TITLE
Model training and evaluation for Mitie and Stanza

### PR DESCRIPTION
**Pull request by gawy**
- added training scripts for mitie and stanza
- single eval script

### CODE REVIEW

File: scripts/train_mitie_ner.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    ## Training
+    if not feature_extractor_path or len(feature_extractor_path.strip()) == 0:
+        # try to download pretrained file
+        feature_extractor_path = os.path.join(workspace_folder, 'total_word_feature_extractor.tokenized.400k.dat')
+
+        if not os.path.exists(feature_extractor_path):
+            url = 'https://dl.dropboxusercontent.com/s/87en1k1cef6wpei/total_word_feature_extractor.tokenized.400k.dat.bz2?dl=0'
+            print(f'Feature extractor file not provided or not found. \nTrying to dowload from {url}')
+            subprocess.run(['curl', url, '--output', feature_extractor_path + '.bz2'])
```
Comment (dchaplinsky) [inline scripts/train_mitie_ner.py:86]: That will work for sure on nix, but ideally you should rely on `requests`/`bz2` lib of python

**Review by dchaplinsky — COMMENTED** [review]
I've mentioned few of typos in PR comments.

Some general feedback (non-mandatory, but nice-to-have):
* Could you please beatify files using `black` python lib, feel free to use 120 characters limit
* Could you please remove commented out debug prints at some stage.

On dropbox links. I think we'll switch to GIT-LFS or similar solution at some stage.

WRT to notebooks. Is it crucial part of the training pipeline or just your drafts and various tries (i.e do I need to read those)?

File: scripts/README.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+
+Run: `scripts/train_stanza_ner.sh`
+
+To use custom word vectors path it with argument `--word_vec` or `-w`. **Stanza uses vectors of dimention `100`**. 
+`scripts/train_stanza_ner.sh -w=path_to_your_wordvec_file`
+
+Vector file first must be converted to .pt format (pytorch binary format) using script from stanza `convert_pretrain.py`. More details please read [here in Stanza docs](https://stanfordnlp.github.io/stanza/word_vectors.html).
+Vector file to be converted - must start with line of format: `<word_count> <dimention [100]>`
+
+Word count can usually be obtained by `wc -l <file>`
```
[the remark this answers is not in the document]
Comment (dchaplinsky) [reply scripts/README.md:32]: Now I can see the link, sorry

File: scripts/README.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+
+The resulting model will reside in `workspace/mitie/mitie_ner_model.dat`
+
+## Stanza
+Stanza already have almost everything that is required to train on our data set merged in.
+Some fixes are pending to be merged from `https://github.com/gawy/stanza.git --branch ner-languk-def-split` (ability to read train-test split from our file insted of random split).
+
+Run: `scripts/train_stanza_ner.sh`
+
+To use custom word vectors path it with argument `--word_vec` or `-w`. **Stanza uses vectors of dimention `100`**.
```
[the remark this answers is not in the document]
Comment (gawy) [reply scripts/README.md:26]: Do you have specific question regarding dimension or it was resolved along with your following question?

File: scripts/train_mitie_ner.py (hunk 1 above)
Comment (gawy) [reply scripts/train_mitie_ner.py]: updated in new version of code

**Comment by gawy** [comment]
- fixed typos
- cleaned up debug print 
- changed to use python requests instead of calling shell scripts
- requirements for pip are not in requirements.txt
- reformatted long lines

File: scripts/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (dchaplinsky) [reply scripts/README.md:26]: I'm mentioning typo here :) dimenSions, not dimentions.

**Review by dchaplinsky — APPROVED** [review]

**Pull request merged**